### PR TITLE
[14.0][FIX] rma_sale: Allow to request a rma from portal order page only with portal user permission

### PR DESCRIPTION
**Pull request by victoralmau**
Allow to request a rma from portal order page only with portal user permission

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa

### CODE REVIEW

**Comment by victoralmau** [comment]
Another related and similar issue: right now it is not possible to generate a rma from the order page without a logged in user, this is correct, right?

**Comment by pedrobaeza** [comment]
>  right now it is not possible to generate a rma from the order page without a logged in user, this is correct, right?

Do you mean having the link with a token? That's correct IMO.

**Review by pedrobaeza — APPROVED** [review]

File: rma_sale/security/ir.model.access.csv [hunk 1]
```diff
@@ -1,3 +1,5 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_sale_order_rma_wizard_user_all,sale.order.rma.wizard.user.all,model_sale_order_rma_wizard,rma.rma_group_user_all,1,1,1,1
+access_sale_order_rma_wizard_portal_all,sale.order.rma.wizard.portal.all,model_sale_order_rma_wizard,base.group_portal,1,1,1,1
```
Comment (chienandalu) [inline rma_sale/security/ir.model.access.csv]: Wouldn't a `sudo` fit this issue better? We're giving portal users whole rights to that model :thinking:

Comment (victoralmau) [reply rma_sale/security/ir.model.access.csv]: This is just about a wizard, do you think this could be a security issue?

**Comment by vincenzo-riggi** [comment]
tested and working

File: rma_sale/security/ir.model.access.csv (hunk 1 above)
Comment (pedrobaeza) [reply rma_sale/security/ir.model.access.csv]: Please change to sudo as commented by chat

**Comment by victoralmau** [comment]
Changes done.

rma-portal

Linked RMAs does not work (rma listing) if no user is logged in.

**Comment by pedrobaeza** [comment]
@chienandalu do you agree?

**Review by chienandalu — APPROVED** [review]
Great :)

**Pull request merged**